### PR TITLE
gmt5: update gmt6 sub-port to version 6.0.0

### PR DESCRIPTION
**Pull request by seisman**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G95
Xcode 10.3 10G8

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] checked your Portfile with `port lint`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by seisman** [comment]
This PR is ready for review. Can anyone help review and merge it?

**Comment by remkos** [comment]
@seisman, if you can make the suggested updates, I can (probably) approve.

**Comment by seisman** [comment]
> * ghostscript is needed in any case during runtime, so it is better to add that as a dependency.

ghostscript is already in the list of `depends_lib`.

> Can you at least add -DGS_ROOT=${prefix} in case of gmt6 (gmt5 does not have this)?

Added for gmt6.

> Furthermore, the cmake configuration has a tendency to mix the library and include files for curl between the system and macports, respectively, which of course is unwanted. I came across this with Fink ([fink/fink-distributions#493](https://github.com/fink/fink-distributions/pull/493)). This can be prevented using:
> -DCURL_INCLUDE_DIR=${prefix}/include
> -DCURL_LIBRARY=${prefix}/lib/libcurl.dylib
> In addition, curl should be added as a dependency.

curl is added to `depends_lib`. I think -DCURL_ROOT is enough, right?

File: science/gmt5/Portfile [hunk 1]
```diff
@@ -8,8 +8,8 @@ version             5.4.5
 revision            1
 conflicts           gmt6
 subport gmt6 {
-    version         6.0.0rc5
-    revision        1
+    version         6.0.0
+    revision        0
```
Comment (mf2k) [inline science/gmt5/Portfile]: In cases like this, port sees `6.0.0rc5` as newer than `6.0.0` and so the `epoch` line needs to be added. 

`epoch   1
`

File: science/gmt5/Portfile [hunk 2]
```diff
@@ -41,15 +40,16 @@ if {${subport} eq "gmt5"} {
...
+                        sha256  8b91af18775a90968cdf369b659c289ded5b6cb2719c8c58294499ba2799b650 \
+                        size    66892468
 }
 
 depends_lib         port:dcw-gmt \
                     port:ghostscript \
                     port:gshhg-gmt \
-                    port:netcdf
+                    port:netcdf \
+                    port:curl
```
Comment (mf2k) [inline science/gmt5/Portfile]: Please add this new dependency in alphabetical order. So put it at the beginning of the list.

**Comment by mf2k** [comment]
The commit message should be:

`gmt5: update gmt6 sub-port to version 6.0.0
`

**Comment by seisman** [comment]
@mf2k Thanks for your review. I've addressed your comments and fixed the commit message.

**Comment by mf2k** [comment]
While the LGTM, this port is not openmaintainer so I want to give @tenomoto more time to look at this.

**Comment by seisman** [comment]
@mf2k It's been two days. Could you merge it?

**Comment by cjones051073** [comment]
Looks fine to me. Merging under maintainer timeout.

**Pull request merged**